release.json

```diff
@@ -41,11 +41,11 @@
         "WINDOWS_DDNPM_SHASUM": "f12af44306eac3ea15828fd12c24d44ae519692a94a0f1f5d4fa868c3e596b07"
     },
     "release-a7": {
-        "INTEGRATIONS_CORE_VERSION": "7.48.0-rc.5",
+        "INTEGRATIONS_CORE_VERSION": "7.48.0-rc.6",
         "OMNIBUS_SOFTWARE_VERSION": "7.48.0-rc.1",
         "OMNIBUS_RUBY_VERSION": "7.48.0-rc.1",
         "JMXFETCH_VERSION": "0.47.10",
-        "JMXFETCH_HASH": "8625f661e34b7180e9df726c69f9edcf71c82682510226948e8a3b37c45c4b9d",
+        "JMXFETCH_HASH": "880c929020d4b79bf1995656d21d9a6859aab3a9460f941eb0b1a6e5502ee4cc",
```

Comment:
:question: Is it expected that the hash changes while the version stays identical?
Shouldn't the change also be present for agent 6?

Reply:
I don't think this should happen, looks like the script issue. I will change that to the original value